WeatherMan.py

```diff
@@ -0,0 +1,33 @@
+import argparse
+import sys
+
+from DataReader import data_reader
+from View import View
+
+weather_report = {}
+
+data_reader = data_reader()
+view = View()
+
+arg_parser = argparse.ArgumentParser()
+arg_parser.add_argument("[report#]", type = int, default = 0, help = 'report number')
```

Comment:
argument should be just `report_type`. using `[report#]` restricts usage only with `getattr`. Similarly it should be `data_dir` not `[data_dir]`